section/OnWindowMessage.cpp

```diff
@@ -0,0 +1,52 @@
+#include "include/global.h"
+
+int __thiscall wxWindow__MSWWindowProc(void *this_, unsigned int uMsg, unsigned int wParam, unsigned long lParam) asm("0x0096D110");
+
+int __thiscall OnWindowMessage(void *this_, unsigned int uMsg, unsigned int wParam, unsigned long lParam) {
+    struct KeyEventInfo {
+        unsigned long repeatCount: 16;
+        unsigned long scanCode: 8;
+        unsigned long isExtended: 1;
+        unsigned long reserved: 4;
+        unsigned long contextCode: 1;
+        unsigned long previousState: 1;
+        unsigned long transitionState: 1;
+
+        constexpr operator unsigned long() const {
```

Comment:
make it function instead of operator and use inline instead.

Reply:
For what? This is a helper structure for specific case: to make the code more understandable and to avoid confusion with bit shifts when filling lParam for the WM_KEYDOWN/UP event https://learn.microsoft.com/ru-ru/windows/win32/inputdev/wm-keydown

The constexpr specifier declares that it is possible to evaluate the value of the function or variable at compile time. A constexpr specifier used in a function or static data member(since C++17) declaration implies inline.

I expect compile-time evaluation; if there were confidence in MinGW compiler's support for consteval, it would have been consteval.

Reply:
I know what `constexpr` is and here it is excessive to use. I've looked through decompiled code and it is the same. And since this structure is just `unsigned long` in a nutshell compiler will evaluate it anyway as const number. I'd keep things simple for now to not confuse people and leave `constexpr` for templates and global values in particular.

Reply:
I can argue a lot about C++. Especially compile time evalutaion since it was in my thesis. But let's keep things simple for thurther changes.